### PR TITLE
Left event support and change default in ServiceRequestState

### PR DESCRIPTION
**Pull request by smunozp**
Handle Left event triggered when there is a call transfer and Agent 1 leaves the call

### CODE REVIEW

**Comment by smunozp** [comment]
the auto format did lots of formatting changes, actual changes where here:
image
the addition of 
image

and change default for :
image

File: lib/src/room.dart [hunk 1]
```diff
@@ -596,14 +679,20 @@ class KurentoRoom extends ChangeNotifier implements Room {
...
     switch (status) {
       case 'REQUEST':
         // Nothing to do.
         break;
 
+      case 'LEFT':
+        _agentName = null;
+        notifyListeners();
+        break;
+
```
Comment (ipainchaud) [inline lib/src/room.dart:695]: This is change 1 of 2 right?

File: lib/src/room.dart [hunk 2]
```diff
@@ -646,12 +736,17 @@ class KurentoRoom extends ChangeNotifier implements Room {
...
+      case 'CANCEL':
         if (_serviceRequestState != ServiceRequestState.ended) {
           _serviceRequestState = ServiceRequestState.ended;
           if (!_isDisposed) notifyListeners();
         }
+        break;
+
+      // All of the other status values.
+      default:
+        _log.warning('ignoring unsupported service request status=$status');
```
Comment (ipainchaud) [inline lib/src/room.dart:749]: line 739 to 749 is change 2 of 2 right? Did I miss something?

File: lib/src/room.dart [hunk 3]
```diff
@@ -646,12 +736,17 @@ class KurentoRoom extends ChangeNotifier implements Room {
         }
         break;
 
-      // All of the other status values -- END, CANCEL, etc. -- are end states.
-      default:
+      case 'END':
+      case 'CANCEL':
```
Comment (ipainchaud) [inline lib/src/room.dart:740]: Did you double check all the possible event to get here? I would like to be sure we haven't forgotten something.

**Comment by ipainchaud** [comment]
your build is failing on a lot of lint issues caused by your auto formatter. Please resolve those: https://app.bitrise.io/build/b369fc28-dbd5-42fb-ac8d-1064cb2ba407

**Comment by ipainchaud** [comment]
can you please update Changelog.md and Pubspec.yaml for the new version?

**Comment by vsomayaji** [comment]
I recommend reverting all of the formatting changes as they're making this hard to review.

You may need to change your line length (**Settings...** > **Editor** > **Code Style** > **Dart** > **Line length**) to `120`.

File: lib/src/room.dart (hunk 3 above)
Comment (smunozp) [reply lib/src/room.dart:729]: i checked with peter and there is no other event that finish a call

**Comment by smunozp** [comment]
redone on 119 to avoid formats changes

**Pull request closed without merging**